examples/vision/integrated_gradients.py

```diff
@@ -0,0 +1,474 @@
+"""
+Title: Integrated Gradients
+Author: [A_K_Nain](https://twitter.com/A_K_Nain)
+Date created: 2020/06/2
+Last modified: 2020/06/2
+Description: How to obtain integrated gradients for a classification model.
+"""
+
+"""
+## Integrated Gradients
+[Integrated Gradients](https://arxiv.org/abs/1701.07875) is a technique for
+attributing a classification model's prediction to its input features. It is
+one of the interpretibility techniques used to understand why did the model make
+this prediction on this input.
+
+[Integrated Gradients](https://arxiv.org/abs/1701.07875) involves following steps:
```

Comment:
These steps may not be very clear for someone not already familiar with the algorithm. Please improve clarity.